build.sh

```diff
@@ -552,6 +552,15 @@ if [[ -n "${DEVICE}" ]]; then
   fi
   # shellcheck disable=SC2012
   [[ -d "${ROOTFS}"/volumio/customPkgs ]] && log "Added custom packages:" "" "$(ls -b "${ROOTFS}"/volumio/customPkgs | tr '\n' ' ')"
+  
+  # Install Volumio ALSA plugins
+  ALSA_DIR_PARENT=$(eval dirname "${ROOTFS}/usr/lib/*/alsa-lib/")
```

Comment:
`ALSA_DIR_PARENT=$(dirname "${ROOTFS}/"usr/lib/*/alsa-lib/)` should be sufficient..

Reply:
When I tested this it didn't work - it left a glob in the result which caused wget to barf

Reply:
Strange - what shell are you using? You could try running it again with `shopt -s failglob`

Reply:
I’m not actually sure. I used the Debian buster (slim) docker image, so probably bash.

Reply:
That should work

> it left a glob in the result which caused wget to barf

if you tired this, it would not work as the glob would fail to expand cause the file doesn't exist
```
wget -nv "${url}" -O "${ROOTFS}"/usr/lib/*/alsa-lib/"libasound_module_pcm_${key}.so"
```

You could instead use the `-P "${ROOTFS}"/usr/lib/*/alsa-lib/` flag and have the remote file aptly named.

But it's a minor detail, if it works, well and good!